packages/harpguru-core/src/components/menu-face/menu-face.tsx

```diff
@@ -0,0 +1,30 @@
+import { View, StyleSheet } from 'react-native'
+import React from 'react'
+
+import { MenuCloseButton } from '../menu-close-button'
+import { getStaticMenuValues } from '../../utils'
+
+type MenuFaceProps = {
```

Comment:
Could this be a `Pick` from the `MenuProps` type

Reply:
Or perhaps it could just *be* the `MenuProps` type so that the props can just be forwarded by the parent without modification. That won't work if the parent handles `children` though.

Reply:
Done